components/proxy/src/test/scala/com/hotels/styx/routing/handlers/ConditionRouterConfigSpec.scala

```diff
@@ -159,7 +159,8 @@ class ConditionRouterConfigSpec extends FunSpec with ShouldMatchers with Mockito
         |""".stripMargin)
 
     val router = new ConditionRouter.ConfigFactory().build(List(), routeHandlerFactory, config)
-    val resp = StyxFutures.await(router.handle(request, HttpInterceptorContext.create(false)).asCompletableFuture())
+
+    val resp = StyxFutures.await(router.handle(request, new HttpInterceptorContext()).asCompletableFuture())
 
     resp.status() should be(BAD_GATEWAY)
   }
```

Comment:
Are the changes in this file still necessary?

Reply:
I put back the old `HttpInterceptorContext.create()` but not `HttpInterceptorContext.create(boolean)` as I assume we do not want to set a precedent of following that convention (see my reply to your other comment of why I think the `create` methods are bad).